tests/acceptance-tests/src/acceptance/http_forwarded_for_test.go

```diff
@@ -0,0 +1,62 @@
+package acceptance_test
+
+import (
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	. "github.com/onsi/gomega/gexec"
+
+	"github.com/cloudfoundry-incubator/cf-test-helpers/cf"
+	"github.com/cloudfoundry-incubator/cf-test-helpers/generator"
+	"github.com/cloudfoundry-incubator/cf-test-helpers/helpers"
+
+	"encoding/json"
+	"net"
+	"strings"
+)
+
+type ResponseJson struct {
+	Header struct {
+		X_Forwarded_For   []string `json:"X-Forwarded-For"`
+		X_Forwarded_Proto []string `json:"X-Forwarded-Proto"`
+	} `json:"Header"`
+}
+
+var headers ResponseJson
+
+var _ = BeforeEach(func() {
+	appName := generator.PrefixedRandomName("CATS-APP-")
+	Expect(cf.Cf(
+		"push", appName,
+		"-b", config.GoBuildpackName,
+		"-p", "../../example_apps/print_request_headers",
+		"-d", config.AppsDomain,
+		"-c", "./bin/debug_app; sleep 1; echo 'done'",
+	).Wait(CF_PUSH_TIMEOUT)).To(Exit(0))
+
+	jsonData := helpers.CurlAppRoot(appName)
+	headers = ResponseJson{}
+	json.Unmarshal([]byte(jsonData), &headers)
+})
+
+var _ = Describe("HTTP forwarded headers are set correctly", func() {
+
+	// Validate that the upstream IP is _not_ on our network
+	It("for X-Forwarded-For", func() {
```

Comment:
Could you restructure this so that it doesn't use comments? It will ensure that the intended behaviour appears in the test output and it makes it less likely that the comment will become out-of-sync with the code. For example:
```
    Describe("X-Forwarded-For", func() {
          It("should have an upstream IP that is not on our network", func() {
```